tests/IntegrationTests.py

```diff
@@ -20,8 +20,10 @@ class IntegrationTests(unittest.TestCase):
     def percy_snapshot(self, name=''):
         if os.environ.get('PERCY_ENABLED', False):
             snapshot_name = '{} - {}'.format(name, sys.version_info)
+
             self.percy_runner.snapshot(
-                name=snapshot_name
+                name=snapshot_name,
+                enable_javascript=True
```

Comment:
Is this line needed? I don't think that it is

Reply:
Removed.